spec/menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'menu'
+
+describe Menu do
+
+  it 'has a list of dishes and prices' do
+    expect(subject).to respond_to(:dishes)
+  end
+
```

Comment:
Can you please explain subject to me - I don't really understand how it works and Anna uses it to.